app/controllers/application_controller.rb

```diff
@@ -1,2 +1,32 @@
 class ApplicationController < ActionController::API
-end
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :user_banned, if: :authenticate_user!
+
+  $bantime = 1.year.from_now
```

Comment:
it would be better to not use global variables, please define those values as constants